healthcheck-sofa-boot-starter/pom.xml

```diff
@@ -39,24 +39,35 @@
         <!--spring-boot-start-->
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot</artifactId>
+            <artifactId>spring-boot-starter</artifactId>
```

Comment:
why do you modify this file?

Reply:
just modify  dependency, it is equivalent